en/datatypes/op.adoc

```diff
@@ -0,0 +1,77 @@
+= Op! datatype
+:toc:
+:numbered:
+
+== Abstract
+
+`Op!` values are infix functions of two arguments that are constructed from existing `native!`, `action!`, `function!`, or `routine!` values.
+
+`Op!` is a member of the following typesets: `any-function!`, `default!`
+
+== Creation
+
+Op values can be created at runtime by using a `make` constructor.
+
+=== Runtime creation
+
+```
+<op> ::= make op! <prefix-function>
+```
+
+*Examples*
+
+Logical `or` and `and` operators.
+
+```red
+>> ||: make op! func [a b][any [a b]]  ; return the first value that is true
+== make op! [[a b]]
+
+>> 1 || 2
+== 1
+
+>> false || 2
+== 2
+
+>> &&: make op! func [a b][all [a b]]  ; if all values are true, return the last value; else return none
+== make op! [[a b]]
+
+>> 1 && 2
+== 2
+
+>> false && 2
+== none
+```
+
+== Evalutaion Steps
+
+`Op!` values take precedence over other function types and over `set-word!` values during evaluation. They cannot contain optional arguments. 
```

Comment:
This should be in the abstract section I think, maybe saying "take evaluation precedence" and remove "during evaluation". 

"contain" is an odd word for opt args. Maybe "take"?